src/subscription_manager/managercli.py

```diff
@@ -1689,20 +1684,29 @@ def _do_command(self):
                     removed_serials = []
                     if self.options.pool_ids:
                         pool_id_to_serials = self.entitlement_dir.list_serials_for_pool_ids(self.options.pool_ids)
-                        success, failure = self._unbind_ids(self.cp.unbindByPoolId, identity.uuid, self.options.pool_ids)
-                        if not success:
-                            return_code = 1
-                        else:
-                            self._print_unbind_ids_result(success, failure, "Pools")
-                            for pool_id in success:
-                                removed_serials.extend(pool_id_to_serials[pool_id])
+                        for pool_id, serials in pool_id_to_serials.iteritems():
+                            success, failure = self._unbind_ids(self.cp.unbindBySerial, identity.uuid, serials)
+                            if success:
+                                self._print_ids(_("Serial numbers successfully"
+                                                  " removed at the server for pool '%s':" % pool_id),
+                                                success)
+                                removed_serials.extend(success)
+                            else:
+                                return_code = 1
```

Comment:
This looks like a case where a generic non-zero return is fine, but if applicable, try to use the error codes in the os module (os.EX_*).